app/controllers/concerns/foreman_remote_execution/hosts_controller_extensions.rb

```diff
@@ -0,0 +1,36 @@
+module ForemanRemoteExecution
+  module HostsControllerExtensions
+    extend ActiveSupport::Concern
+    include Foreman::Renderer
+
+    included do
+      alias_method_chain(:action_permission, :remote_execution)
+    end
+
+    def reprovision
+      find_resource
+      script_template = @host.provisioning_template(:kind => 'script')
+      if script_template.nil?
+        process_error :redirect => :back, :error_msg => _("No script provisioning template available")
```

Comment:
Single quote string literals (and in a bunch of other places) :smile:  rubocop can fix all of these.